src/items.rs

```diff
@@ -790,6 +737,55 @@ fn format_impl_ref_and_type(context: &RewriteContext,
     }
 }
 
+// Returns false if failed to update result: then, try using multiline.
+fn format_trait_ref_then_update_result(context: &RewriteContext,
```

Comment:
Thanks for your feedback!
No real reason, I was being lazy here.
I would like to change this to return `Option<String>`.